lib/routes/devices-and-sessions.js

```diff
@@ -63,6 +65,41 @@ module.exports = (log, db, config, customs, push, devices) => {
     return result
   }
 
+  function marshallLocation (location, request) {
+    let language
+
+    try {
+      const languages = i18n.parseAcceptLanguage(request.app.acceptLanguage)
+      language = i18n.bestLanguage(languages, supportedLanguages, defaultLanguage)
+
+      if (language[0] === 'e' && language[1] === 'n') {
```

Comment:
I've assumed it is safe to treat all languages beginning with `en...` as English. Maybe there's a more official way to do this test?

Reply:
I am not sure either, but I think this is ok for now?